src/frequenz/sdk/actor/_power_managing/_power_managing_actor.py

```diff
@@ -47,6 +48,8 @@ def __init__(  # pylint: disable=too-many-arguments
         Args:
             component_category: The category of the component this power manager
                 instance is going to support.
+            component_type: The type of the component of the given category that this
+                actor is responsible for.
```

Comment:
Added a few lines.

Reply:
Although the API exposes component type for all categories, the client exposes it only for inverters, because that's been the only thing we've needed in the SDK.